RELEASE.md

```diff
@@ -0,0 +1,24 @@
+# Steps to issue a new release
+
+Before starting the release process:
+
+- Merge all PRs to `main`
+- Make sure you've updated the version number in TBD
```

Comment:
@chrisjkuch - where do we need to update the version in the code?

Reply:
In the `pyproject.toml` file
```suggestion
- Make sure you've updated the version number in `pyproject.toml`
```